spec/requests/comments_create_spec.rb

```diff
@@ -93,4 +93,79 @@ def comment_params(params = {})
       expect(response.parsed_body["error"]).to be_present
     end
   end
+
+  context "when there's a notification alredy for comment" do
```

Comment:
@msarit thanks! Since CI passes, I believe the problem to be specific to your setup. My guess is that  database cleaning doesn't work. `rake db:test:prepare` might help with that.